fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalCatalog.java

```diff
@@ -149,6 +150,9 @@ public abstract class ExternalCatalog
     protected Optional<Boolean> useMetaCache = Optional.empty();
     protected MetaCache<ExternalDatabase<? extends ExternalTable>> metaCache;
 
+    protected IdentifierMapping identifierMapping;
+    private boolean mappingsInitialized = false;
```

Comment:
Why using a separator flag `mappingsInitialized`?
Why not just using `initialized`?